### DISCUSSION TITLE
Carbon Hack 24

### DISCUSSION THREAD

**Discussion by NAMRATA-WOKE** [Green-Software-Foundation/discussions#117] [Events]
![Twitter-cover (1)](https://github.com/Green-Software-Foundation/discussions/assets/114516498/639bb39b-9b3f-45e9-af4e-a83b210db2fe)

**Carbon Hack will take place from Monday, March 18 to Monday, April 8.** 

This year, we're pushing boundaries by going beyond carbon—exploring the full spectrum of environmental impacts with the Impact Framework, an open source Beta measurement tool for measuring software across all components and environments, to reduce the ecological footprint of software.

NEXT STEPS

1. [Sign up today](https://hack.greensoftware.foundation/) to get more information about the hackathon and Impact Framework
2. [Attend the next live Prep and Q&A Session](https://grnsft.org/hack/live) with GSF Executive Director Asim Hussain to learn more about what's in store and how you can participate this year.

**Comment by mheaversmozilla** [comment]
@NAMRATA-WOKE - [posted this here as well](https://github.com/Green-Software-Foundation/hack/discussions/35), but in case this is a better spot, I'm trying to get folks from my company, Mozilla, to participate in the Hackathon for Earth Week (month / century?). I think approaching them wth a specific challenge to solve might make it easier to recruit - so I'm thinking this might be one that would make sense to try and solve. We've done a lot of work around CO2e measurement within the Firefox Profiler in collaboration with the Green Web Foundation. Curious your thoughts here. Does this seem like a good lane for folks from Mozilla if I can get them to participate? Or is there something else more pressing / relevant that comes to mind. Let me know if there's a better venue for this question.

**Reply by NAMRATA-WOKE** [reply]
@mheaversmozilla Thank you for thinking about ways to mobilize folks at Mozilla to participate in the hackathon. I'm looping in our two lead developers of Impact Framework team who might be better positioned to guide you: @jawache @jmcook1186 

In terms of pressing needs, we'd really love to see participants build plugins within IF to convert observable metrics into impact data. More on that here: https://if.greensoftware.foundation/

The IF team has developed a training workshop to educate participants on how to work with IF: https://grnsft.org/hack/workshop-recording

Hope this helps! Sorry for keeping you waiting here!
